after_commit_everywhere.gemspec

```diff
@@ -34,6 +34,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "bundler", "~> 2.0"
   spec.add_development_dependency "isolator", "~> 0.7"
   spec.add_development_dependency "pry"
+  spec.add_development_dependency "nokogiri", "~> 1.13.3"
```

Comment:
I don't think that we need to specify it explicitly. `bundle update` will update it in the lockfile.
```suggestion
```